packages/main/src/Breadcrumbs.ts

```diff
@@ -154,7 +154,7 @@ class Breadcrumbs extends UI5Element {
 	 * @public
 	 */
 	@property({ type: BreadcrumbsSeparatorStyle, defaultValue: BreadcrumbsSeparatorStyle.Slash })
-	separatorStyle!: `${BreadcrumbsSeparatorStyle}`;
+	separators!: `${BreadcrumbsSeparatorStyle}`;
```

Comment:
I think renaming the enum also makes sense, BreadcrumbsSeparatorStyle -> BreadcrumbsSeparator.